docs/ds9.rst

```diff
@@ -0,0 +1,82 @@
+.. _gs-ds9:
+
+Reading/writing to DS9 region files
+===================================
+
+The regions package provides functions to serialise and de-serialise Python lists of
+`~regions.Region` objects to DS9 region strings: `~regions.ds9_objects_to_string`
+and `~regions.ds9_string_to_objects`.
+
+.. code-block:: python
+
+    >>> from regions import ds9_objects_to_string, ds9_string_to_objects
+    >>> reg_string = 'galactic\ncircle(42,43,3)'
+    >>> regions = ds9_string_to_objects(reg_string)
+    >>> regions[0]
+    CircleSkyRegion
+    center: <Galactic Coordinate: (l, b) in deg
+        (42.0, 43.0)>
+    radius: 3.0 deg
+    >>> ds9_objects_to_string(regions, coordsys='galactic')
+    '# Region file format: DS9 astropy/regions\ngalactic\ncircle(42.0000,43.0000,3.0000)\n'
+
+There's also `~regions.write_ds9` and `~regions.read_ds9` with write to and read from
+a file in addition to doing the region serialisation and parsing.
+
+.. code-block:: python
+
+    >>> from regions import read_ds9, write_ds9
+    >>> filename = 'ds9.reg'
+    >>> write_ds9(regions, filename)
+    >>> regions = read_ds9(filename)
+    >>> regions
+    [CircleSkyRegion
+     center: <FK5 Coordinate (equinox=J2000.000): (ra, dec) in deg
+         (245.3477, 24.4291)>
+     radius: 3.0 deg]
+
+Often DS9 region files contain extra information about colors or other attributes.
+This information is lost when converting to `~regions.Region` objects.
```

Comment:
This should not be true; the data are (or at least were) preserved in `.meta` and `.visual`